src/components/Template.js

```diff
@@ -50,7 +46,7 @@ class Template extends React.Component {
 
     return (
       <div
-        {...otherProps}
```

Comment:
This is where data={} was sent. In dev it was ok because we did omit using the propTypes, but in production it was not because propTypes are removed from the production builds. Thus doing an omit() with an empty object